### ISSUE TITLE
Code block display problem on smartphone and reduced window

### ISSUE THREAD

**Issue by marcelo57** [riggraz/no-style-please#17]
hello, I really like this theme, but code blocks seem to mess with de display of the page if you view it from a smartphone or a reduced window.

half of the page dissapears into a white screen (maybe it is a issue with the dark mode?)
it happens in the demo page too:

![photo_2021-03-16_03-44-21](https://user-images.githubusercontent.com/78581471/111266961-07960800-860a-11eb-91f2-7e8dea0175c6.jpg)

I'm just starting to learn about all this stuff, hope its not my bad...
thanks!

**Comment by riggraz** [comment]
I think the problem is due to the CSS rule `width: fit-content` in the wrapper div. I'll take a look into it and fix it

**Comment by riggraz** [comment]
I investigated a bit and that's what I found.

First of all, the problem has nothing to do with dark mode since it also occurs in light mode. Moreover, in my tests this problem occured only in Chrome and Safari.

It has to do with `<code>` tags that don't seem to work as expected. All other elements (images, blockquotes, tables, ...) respect the size of the parent wrapper div (`<div class="w">`), but `<code>` doesn't. It takes up all the width necessary to render its longest line, up to a max width of 640px (defined again in the wrapper div).

This surely has something to do with the wrapper div style: `width: fit-content`. Removing that line solves the problem, but brings some inconveniences so I don't want to change it.

Instead I added `white-space: pre-wrap` to the style of `<code>` elements. This solves the problem but has the incovenience that now lines in <code> blocks wrap if they're too long. Not the best imo but it's okay for now.
